deployment/updates/scripts/xml/2022_11_20_updateUniqueKmsNewItemPendingModeration2.template.xml

```diff
@@ -0,0 +1,114 @@
+                            <code>$scope->getObject()->getEntryId()</code>
+                        </value>
+                    </item>
+                    <item objectType="KalturaEventNotificationParameter">
+                        <key>entry_name</key>
+                        <description>Entry name</description>
+                        <value objectType="KalturaEvalStringField">
+                            <code>($entry = entryPeer::retrieveByPk($scope->getObject()->getEntryId())) ? $entry->getName() : ''</code>
+                        </value>
+                    </item>
+                    <item objectType="KalturaEventNotificationParameter">
+                        <key>category_id</key>
+                        <description>Category ID</description>
+                        <value objectType="KalturaEvalStringField">
+                            <code>$scope->getObject()->getCategoryId()</code>
+                        </value>
+                    </item>
+                    <item objectType="KalturaEventNotificationParameter">
+                        <key>category_name</key>
+                        <description>Category name</description>
+                        <value objectType="KalturaEvalStringField">
+                            <code>($category = categoryPeer::retrieveByPk($scope->getObject()->getCategoryId())) ? $category->getName() : ''</code>
+                        </value>
+                    </item>
+                    <item objectType="KalturaEventNotificationParameter">
+                        <key>category_name_encoded</key>
+                        <description>Encoded category name</description>
+                        <value objectType="KalturaEvalStringField">
+                            <code>urlencode(($category = categoryPeer::retrieveByPk($scope->getObject()->getCategoryId())) ? $category->getName() : '')</code>
+                        </value>
+                    </item>
+                    <item objectType="KalturaEventNotificationParameter">
+                        <key>creator_id</key>
+                        <description>Entry creator ID</description>
+                        <value objectType="KalturaEvalStringField">
+                            <code>($entry = entryPeer::retrieveByPk($scope->getObject()->getEntryId())) ? $entry->getFromCustomData('creatorPuserId') : ''</code>
+                        </value>
+                    </item>
+                    <item objectType="KalturaEventNotificationParameter">
+                        <key>creator_name</key>
+                        <description>Entry creator name</description>
+                        <value objectType="KalturaEvalStringField">
+                            <code>($entry = entryPeer::retrieveByPk($scope->getObject()->getEntryId())) ? ($kuser = kuserPeer::retrieveByPk($entry->getCreatorKuserId())) ? $kuser->getFirstName() . ' ' .  $kuser->getLastName(): '' : ''</code>
```

Comment:
would be nicer to have the condition with (), the last part where you have : '' :'' can be a bit confusing

Reply:
Fixed.